pom.xml

```diff
@@ -109,22 +117,22 @@
         <dependency>
             <groupId>org.elasticsearch</groupId>
             <artifactId>elasticsearch</artifactId>
-            <version>2.1.1</version>
+            <version>2.4.6</version>
         </dependency>
 
         <!-- Elastic search multicast, used for discovering elastic search  instances-->
         <dependency>
             <groupId>org.elasticsearch.plugin</groupId>
             <artifactId>discovery-multicast</artifactId>
-            <version>2.1.1</version>
+            <version>2.4.6</version>
         </dependency>
 
 
         <!-- Ical4j-->
         <dependency>
             <groupId>org.mnode.ical4j</groupId>
             <artifactId>ical4j</artifactId>
-            <version>2.0.5</version>
+            <version>2.2.7</version>
         </dependency>
 
         <!-- RSS Feeds -->
```

Comment:
It looks like `rome` was moved to `com.rometools`, so it may be worth looking into upgrading it.

Reply:
updated to use com.rometools